Source/Scene/Cesium3DTileset.js

```diff
@@ -1751,6 +1751,8 @@ Cesium3DTileset.prototype.loadTileset = function (
     throw new RuntimeError("The tileset must be 3D Tiles version 0.0 or 1.0.");
   }
 
+  Cesium3DTileset.checkSupportedExtensions(this._extensions);
```

Comment:
A few things here:

1. `this._extensions` is not yet initialized (see lines 965 and 974) at this point in the code, as `loadTileset()` gets called first. Use `tilesetJson.extensionsRequired` instead.
2. Also it would be good to check `if (defined(tilesetJson.extensionsRequired)) { /* check supported extensions */ }` just in case the array is undefined (which will be the case most of the time)
3. To answer your question about whether we need to add `this._extensionsRequired`, I don't think we need it, as this is the only place the required extensions list is used.